<h1>Open Tree of Life reference taxonomy version 2.9</h1>

<p>Version 2.9 draft 12 was generated on 13 October 2015.</p>

<p>This index.html file is obsolete.  Please
  consult <a href="https://github.com/OpenTreeOfLife/reference-taxonomy/blob/master/doc/ott2.9.md">the
  release notes in github</a>.</p>

<h2>Download</h2>

<p><a href="http://files.opentreeoflife.org/ott/ott2.9/ott2.9.tgz">Download</a> (gzipped tar file, 91 Mbyte) </p>

<h2>Contents</h2>

<p>All files are encoded UTF-8.  For documentation about file formats, see <a href="https://github.com/OpenTreeOfLife/reference-taxonomy/wiki/Interim-taxonomy-file-format">the documentation in the reference taxonomy
wiki</a>,
on github.</p>

<p><strong>taxonomy.tsv</strong>: The file that contains the taxonomy.</p>

<p><strong>synonyms.tsv</strong>: The list of synonyms.</p>

<p><strong>conflicts.tsv</strong>: Report on taxa from input taxonomies that are
  hidden because they are paraphyletic with respect to a higher
  taxon from a higher priority input taxonomy.  Number in first column is depth in taxonomic tree of
  nearest common ancestor of its children.</p>

<p><strong>deprecated.tsv</strong>: List all taxon ids occurring in phylesystem studies that have been deprecated since previous version. </p>

<p><strong>differences.tsv:</strong> List of differences between ott2.9 and ott2.8.</p>

<p><strong>otu_differences.tsv:</strong> List of differences between ott2.9 and ott2.8 that involve taxa in phylesystem.</p>

<p><strong>hidden.tsv</strong>: Report on 'hidden' taxa (incertae sedis and other suppressed taxa). Columns are OTT id, name, source taxonomy and id, containing major group, and flags (reasons for hiding).</p>

<p><strong>log.tsv</strong>: Debugging information related to homonym resolution.</p>

<p><strong>version.tsv</strong>: The version of OTT.</p>

<h2>Build script</h2>

<p>The reference taxonomy is an algorithmic combination of several
source taxonomies.  For code,
see <a href="https://github.com/OpenTreeOfLife/reference-taxonomy">the
source code repository</a>.
Version 2.9 draft 12 was generated using 
<a href="https://github.com/OpenTreeOfLife/reference-taxonomy/commit/a58f25af5a35c988684979c87f156a5e9364c54d">commit a58f25a</a>.</p>

<p>Where taxonomies conflict regarding taxon relationships, they are
resolved in favor of the higher priority taxonomy.  The priority
ordering is as given below.</p>

<h2>Sources</h2>

<p>Any errors in OTT
should be assumed to have been introduced by the Open Tree of Life 
project until confirmed as originating in the source taxonomy.</p>

<p>Download locations are for the particular versions used to construct
OTT 2.9.  For new work, current versions of these sources should be
retrieved.</p>

<ol>
<li><p>Taxonomy from: 
DS Hibbett, M Binder, JF Bischoff, M Blackwell, et al. 
A higher-level phylogenetic classification of the <i>Fungi</i>.
<a href="http://dx.doi.org/10.1016/j.mycres.2007.03.004">Mycological Research</i> <b>111</b>(5):509-547, 2007</a>.
Newick string with revisions
archived at <a href="http://figshare.com/articles/Fungal_Classification_2015/1465038">http://figshare.com/articles/Fungal<em>Classification</em>2015/1465038</a>.
<br />
Download location: <a href="http://purl.org/opentree/ott/??TBD??">http://purl.org/opentree/ott/??TBD??</a></p></li>
<li><p>Taxonomy from: SILVA 16S ribosomal RNA database, version 115.
See: Quast C, Pruesse E, Yilmaz P, Gerken J, Schweer T, Yarza P, Peplies J,
Gl&ouml;ckner FO (2013) The SILVA ribosomal RNA gene database project:
improved data processing and web-based tools. 
<a href="http://dx.doi.org/10.1093/nar/gks1219">Nucleic Acids Research</i> 41 (D1): D590-D596</a>.
Web site: <a href="http://www.arb-silva.de/">http://www.arb-silva.de/</a>.
<br />
Download location: <a href="ftp://ftp.arb-silva.de/release_115/Exports/tax_ranks_ssu_115.csv">ftp://ftp.arb-silva.de/release_115/Exports/tax_ranks_ssu_115.csv</a>.</p></li>
<li><p>Index Fungorum.
Download location: derived from database query result files provided by Paul
Kirk, 7 April 2014 (personal communication).
Web site: <a href="http://www.indexfungorum.org/">http://www.indexfungorum.org/</a>.
<br />
Download location (converted to OTT format): <a href="http://purl.org/opentree/ott/??TBD??">http://purl.org/opentree/ott/??TBD??</a>.</p></li>
<li><p>Taxonomy from:
Sch&auml;ferhoff, B., Fleischmann, A., Fischer, E., Albach, D. C., Borsch,
T., Heubl, G., and M&uuml;ller, K. F. (2010). Towards resolving Lamiales
relationships: insights from rapidly evolving chloroplast
sequences. 
<a href="http://dx.doi.org/10.1186/1471-2148-10-352"><i>BMC evolutionary biology</i> 10(1), 352.</a>.
Manually transcribed from the paper and converted to OTT format.
<br />
Download location: <a href="http://purl.oeg/opentree/ott/ott2.8/inputs/lamiales-20140118.tsv">http://purl.org/opentree/ott/ott2.8/inputs/lamiales-20140118.tsv</a></p></li>
<li><p>WoRMS - harvested from web site using web API over two weeks ending around 4 August 2014.
<a href="http://www.marinespecies.org/aphia.php">http://www.marinespecies.org/aphia.php</a></p></li>
<li><p>NCBI Taxonomy, from the 
<a href="http://www.ncbi.nlm.nih.gov/">US National Center on Biotechnology Information</a>.
Web site: <a href="http://www.ncbi.nlm.nih.gov/Taxonomy/">http://www.ncbi.nlm.nih.gov/Taxonomy/</a>.
Current version download location:
<a href="ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump.tar.gz">ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump.tar.gz</a>
<br />
For OTT 2.9 we used a version downloaded from NCBI on 27 June 2015.
Download location: <a href="http://purl.org/opentree/ott/??TBD??">http://purl.org/opentree/ott/??TBD??</a>.
</li></p></li>
<li><p>GBIF Backbone Taxonomy, from the 
<a href="http://www.gbif.org/">Global Biodiversity Information facility</a>.
Current version download location: 
<a href="http://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c">http://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c</a>.
<br />
We used a version dated 2013-07-02.
Download location: <a href="http://purl.org/opentree/gbif-backbone-2013-07-02.zip">http://purl.org/opentree/gbif-backbone-2013-07-02.zip</a>.</p></li>
<li><p><a href="http://www.obis.org.au/irmng/">Interim Register of Marine and Nonmarine Genera (IRMNG)</a>, from CSIRO.
Current version download location:
<a href="http://www.cmar.csiro.au/datacentre/downloads/IRMNG_DWC.zip">http://www.cmar.csiro.au/datacentre/downloads/IRMNG_DWC.zip</a>.
<br />
We used a version dated 2014-01-31.  Download location:
<a href="http://purl.org/opentree/ott/ott2.8/inputs/IRMNG_DWC-2014-01-30.zip">http://purl.org/opentree/ott/ott2.8/inputs/IRMNG_DWC-2014-01-30.zip</a>.</p></li>
<li><p>Taxon identifiers are carried over from <a href="http://purl.org/opentree/ott/ott2.8/">OTT 2.8</a> when possible</p></li>
</ol>

<p>The non-Fungi content of Index Fungorum is separated from the Fungi
content and given a priority lower than NCBI but higher than GBIF.</p>

<p>The non-Malacostraca content of WoRMS is separated from the
Malacostraca content and given a priority lower than NCBI but higher
than GBIF.</p>

<h2>Release notes</h2>

<p>Changes since OTT 2.8 (a.k.a 2.8draft5) which was built on 11 June 2014:</p>

<p>Statistics:</p>

<ul>
<li>Identifiers: 3461408</li>
<li>Visible: TBD</li>
<li>Synonyms: 910144</li>
<li>In deprecated file (used in phylesystem): 2889</li>
<li>In deprecated file (used in synthesis): 309</li>
<li>Taxa dissolved due to conflict (conflicts.tsv): 1319</li>
</ul>

<p>New flags:</p>

<ul>
<li>placed - similar to incertae_sedis (this means a child of an
inconsistent taxon, where t is inconsistent if it occurs in a
lower-priority taxonomy but is inconsistent with the higher-priority
taxonomies.  'tattered' is now deprecated)</li>
<li>placed_inherited - descends from a placed taxon.</li>
<li>inconsistent (formerly 'tattered') - taxon in lower priority 
taxonomy that is inconsistent (see above).</li>
<li>merged - this taxon was consistent with another and got folded 
into it.  Taxon is hidden, children aren't.  Taxon may be
revived if it's learned later that the it is actually different.</li>
<li>was<em>container - treat same as incertae</em>sedis, merged, and
inconsistent - this was formerly a 'bucket' but is now empty and is
preserved as a placeholder.</li>
<li>extinct - replaces extinct_direct.</li>
<li>major<em>rank</em>conflict - replaced major<em>rank</em>conflict_direct.</li>
<li>sibling_lower is deprecated, that information in not recorded (but you can
always tell, just by looking at ranks of the siblings).  sibling_higher
is retained.</li>
<li>Deprecated: tattered, tattered_inherited</li>
</ul>

<p>Specific content changes:</p>

<ul>
<li>Added WoRMS</li>
<li>Updated Hibbett 2007 from <a href="http://figshare.com/articles/Fungal_Classification_2015/1465038">http://figshare.com/articles/Fungal_Classification_2015/1465038</a></li>
<li>Minor IF update (to 7 April 2014 and modified processing software)</li>
<li>Minor GBIF update (same origin content, modified processing, much faster)</li>
<li>NCBI update (27 June 2015)</li>
<li>Fixes for many bugs reported in feedback and reference-taxonomy repos (see milestones)</li>
</ul>

<p>Generic content changes (processing):</p>

<ul>
<li>'Lumping' is now allowed more promiscuously than before</li>
<li>uniqueName now shows highest possible discriminating node</li>
<li>Somewhat more informative deprecated.tsv</li>
<li>'skeleton' feature replaces 'pinning' for homonym separation (see
tax/skel/ for list of barrier nodes)</li>
<li>"unique names" show highest distinguishing taxon, e.g. "Morganella
(genus in kingdom Fungi)" instead of "Morganella (genus in family
Agaricaceae)"</li>
</ul>

<p>Jython API changes:</p>

<ul>
<li>Experimental 'claim' system, an implementation of what JAR described in
his IEvoBio 2014 talk.  See util/claim.py in the repo.</li>
<li>.add method is deprecated (use .take instead)</li>
</ul>

<p>Other:</p>

<ul>
<li>SKOS generation</li>
<li>Homonym reports</li>
</ul>
